### PR TITLE
feat(type): add type definition for mysql.count method

### PR DESCRIPTION
**Pull request by Krryxa**
##### Checklist

- [ ] `npm test` passes
- [ ] commit message follows commit guidelines

##### Description of change

add type definition for mysql.count method

### CODE REVIEW

**Review by fengmk2 — APPROVED** [review]

File: index.d.ts [hunk 1]
```diff
@@ -60,6 +60,8 @@ interface EggMySQL {
     where?: object,
     condition?: EggMySQLCondition
   ): Promise<EggMySQLGetResult>;
+  /// get counts from table 
+  count: (table: string, values: object) => Promise<number>;
```
Comment (fengmk2) [inline index.d.ts:64]: values should name to where, keep the same as the real count method https://github.com/ali-sdk/ali-rds/blob/master/lib/operator.js#L62

```js
count: (table: string, where?: object)
```

Comment (Krryxa) [reply index.d.ts]: done

**Comment by Krryxa** [comment]
image
how can I fix this?
Can you ignore it and merge

**Comment by fengmk2** [comment]
@Krryxa I will fix mocha missing in Node.js v14

**Pull request merged**